instrumentation/opentelemetry/span.go

```diff
@@ -31,6 +31,15 @@ func (l *AttributeList) GetValue(key string) interface{} {
 	return nil
 }
 
+func (l *AttributeList) GetAll() []sdk.Attribute {
```

Comment:
@varkey98 If this is for the purpose of being able to access all attributes in filter impl, can we find a way to just expose a way to iterate over each attribute in the list? Rather than incurring overhead of another slice and copying over to that slice. All attributes are already contained in the list.

Reply:
Got it, let me try. I created a new type because of the generic nature of `AttributeList`